### DISCUSSION TITLE
Questions about finetuning (our model)

### DISCUSSION THREAD

**Discussion by nihil39** [ACEsuit/mace#742] [Q&A] [answered]
Hi, 

we have trained a MACE model for pure water with the idea of adding configurations with water and ions in solution. 

Is it possible to [finetune](https://mace-docs.readthedocs.io/en/latest/guide/finetuning.html) our model or this possibility is reserved only for foundation models? 

Thanks

Edit: 

Sorry I read the guide again and reading this _If you want to finetune another model, the model will be loaded from the path provided --foundation_model=$path_model, but you will need to provide the full set of hyperparameters (hidden irreps, r_max, etc.) matching the model._ it seems to be possible to finetune a different model and not only the foundation one. 

What are the hyperparameters to be provided to the model? I think only the one we modified from the default values are needed but I'm not sure

**Comment by ilyes319** [comment, marked as answer]
Hey yes indeed you can. Actually, you do not need to pass any hyper anymore, the doc is oudated, I will have to change it. Just put the path to your model in `foundation_model=$path_model` and that should work!
